go.mod

```diff
@@ -1,120 +1,60 @@
 module github.com/giantswarm/azure-admission-controller
 
-go 1.17
+go 1.16
```

Comment:
from 1.17 to 1.16?

Reply:
you mean during CI? I've seen this, but normally it only happens the first time locally, then I never see it again after running `go mod tidy -go=1.16 && go mod tidy -go=1.17`.

Reply:
yeah it made the CI fail. I can try again

Reply:
We are using 1.18 in circleci, maybe worth trying that? I mean upgrading locally, running go mod tidy locally and pushing that.

Reply:
yeah, that worked :bow: